website/docs/configurations.md

```diff
@@ -197,7 +197,10 @@ Options useful for reading tables via `read.format.option(...)`
 
 ### Write Options {#Write-Options}
 
-You can pass down any of the WriteClient level configs directly using `options()` or `option(k,v)` methods.
+Hudi supports [parquet modular encryption](/docs/encryption) and [parquet bloom filters](/docs/parquet_bloom) through hadoop configurations.
+
```

Comment:
Can we add a four-level heading for this new configuration part, I mean it deserves a separate paragraph for the introduction.

Reply:
added parquet_config heading